pytorch_lightning/loggers/base.py

```diff
@@ -395,12 +406,15 @@ def __init__(self):
         self._experiment = DummyExperiment()
 
     @property
+    @rank_zero_experiment
```

Comment:
I recommend not adding it to the base class, as it is not strictly a requirement. 
Besides, the decorator does not get inherited anyway